skimage/measure/fit.py

```diff
@@ -573,6 +573,10 @@ def ransac(data, model_class, min_samples, residual_threshold,
         where the probability (confidence) is typically set to a high value
         such as 0.99, and e is the current fraction of inliers w.r.t. the
         total number of samples.
+    exceptions : exception class or tuple of exception classes
```

Comment:
I am afraid of hiding bugs by suppressing *any* exception.  But perhaps it's not worth worrying about it too much for now--the user at least has the option of setting the exception set to empty.